ROADMAP.md

```diff
@@ -9,46 +9,38 @@ Any dates listed below and the specific issues that will ship in a given milesto
 See the [Github project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
 
-## Rook 1.6
+## Rook 1.7
 
-The following high level features are targeted for Rook v1.6 (March 2021). For more detailed project tracking see the [v1.6 board](https://github.com/rook/rook/projects/20).
+The following high level features are targeted for Rook v1.7 (July 2021). For more detailed project tracking see the [v1.7 board](https://github.com/rook/rook/projects/21).
 
 * Ceph
-  * Support the Ceph Pacific release
-  * CephFS Mirroring support for data replication across clusters
-  * RBD mirroring configuration between clusters to support application DR (disaster recovery)
+  * Helm chart for the cluster CR [#2109](https://github.com/rook/rook/issues/2109)
```

Comment:
We also could consider doing larger helm re-design work: https://github.com/rook/rook/issues/6776

Reply:
Honestly, I'm not sure we need to do more helm re-design work. IMO a chart for the cluster CR fills the gap, but let's discuss more.